InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -7614,7 +7613,7 @@ private string iwParseString2ListSub(string str, int trimString, int doCapitaliz
 
             str = iwParseString2ListSub(str, trimString, doCapitalize);
 
-            if (str != "" || keepNulls == true)
+            if (!String.IsNullOrEmpty(str) || keepNulls == true)
```

Comment:
By this point str cannot be null for two reasons: `iwParseString2ListSub` now cannot return a null, and str would have already caused a null pointer exception if it was.